test/e2e/warnings_test.go

```diff
@@ -77,18 +77,18 @@ spec:
 		kapp.RunWithOpts([]string{"deploy", "-f", "-", "-a", crdName},
 			RunOpts{StdinReader: strings.NewReader(yaml)})
 	})
-	logger.Section("deploying with --warnings flag", func() {
+	logger.Section("deploying without --warnings flag", func() {
 		yaml := strings.Replace(crYaml, "<cr-name>", "cr-1", 1)
-		out, _ := kapp.RunWithOpts([]string{"deploy", "-f", "-", "-a", crName, "--warnings=true"},
+		out, _ := kapp.RunWithOpts([]string{"deploy", "-f", "-", "-a", crName},
 			RunOpts{StdinReader: strings.NewReader(yaml)})
 
 		if !strings.Contains(out, customWarning) {
```

Comment:
should we assert on full output to see where warnings are coming up?

Reply:
Updated the test cases